src/test/scala/nl.knaw.dans.easy.deposit/SubmitterSpec.scala

```diff
@@ -37,17 +40,37 @@ class SubmitterSpec extends TestSupportFixture with MockFactory {
   private val doi = datasetMetadata.doi
     .getOrElse(fail("could not get DOI from test input"))
 
-  "submit" should "write all files" in {
+  "lazy constructor" should "not be called by EasyDepositApiApp if the configured group does not exist" in {
+    val props = minimalAppConfig.properties
+    props.setProperty("deposit.permissions.group", "not-existing-group")
+
+    Try(new EasyDepositApiApp(new Configuration("", props))) should matchPattern {
```

Comment:
I think you can do something like:

```scala
the [IOException] thrownBy(new EasyDepositApiApp(new Configuration("", props))) should have message "Group not-existing-group could not be found"
```

might be more descriptive of what you're actually testing here